test/e2e/events.go

```diff
@@ -76,7 +69,7 @@ var _ = Describe("Events", func() {
 			Failf("Failed to create pod: %v", err)
 		}
 
-		expectNoError(waitForPodRunning(c, pod.Name))
+		expectNoError(framework.WaitForPodRunning(pod.Name))
```

Comment:
optional nit: Can you please log running pods on the node when this fails and has a node (eg: the capacity exceeded case)?